easybuild/easyconfigs/s/supernova/supernova-2.1.1.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Tarball'
+
+name = 'supernova'
+version = '2.1.1'
+
+homepage = 'https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome'
+description = """Supernova is a software package for de novo assembly from Chromium Linked-Reads
+ that are made from a single whole-genome library from an individual DNA source"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# no direct download
+# https://support.10xgenomics.com/de-novo-assembly/software/downloads/latest
+sources = ['supernova-%(version)s.tar.gz']
+checksums = ['2f58eb66951e257b89359134ab8e35ad638c4ed51cb3fb8121625dfcc7761938']
+keepsymlinks = True
+
+sanity_check_paths = {
+    'files': ['supernova'],
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': "",  # add the installation dir to PATH
+}
+
+sanity_check_commands = ['supernova testrun --id=tiny']
```

Comment:
my test seems to be stalled here, no cpu usage...

(also, wouldn't this be better in `tests` instead of `sanity_check`?)

Reply:
@migueldiascosta The `test` step is done prior to the `install` step, so `supernova` won't be there yet...

Reply:
@boegel I meant the test cases, using the `tests` easyconfig parameter, those are run after install, e.g. `tests = ['%(installdir)s/supernova testrun --id=tiny']`?

Reply:
That's true, but if those fail, the module is installed anyway...